guides/common/modules/con_provisioning-methods-in-project.adoc

```diff
@@ -3,29 +3,28 @@
 [id="Provisioning-Methods-in-{ProjectNameID}_{context}"]
 = Provisioning methods in {ProjectName}
 
-With {ProjectName}, you can provision hosts by using the following methods.
+With {ProjectName}, you can provision hosts by using the following provisioning methods.
 
-Bare-metal hosts::
-{Project} provisions bare-metal hosts primarily by using PXE boot and MAC address identification.
-When provisioning bare-metal hosts with {Project}, you can do the following:
+Network based::
+You can use this method with hosts that provide PXE booting or UEFI HTTP booting.
 +
-* Create host entries and specify the MAC address of the physical host to provision.
-* Boot blank hosts to use the {Project} Discovery service, which creates a pool of hosts that are ready for provisioning.
-* Provision hosts with UEFI Secure Boot.
-ifndef::satellite[]
-* Boot and provision hosts by using PXE-less methods.
-endif::[]
+You are required to configure the necessary network booting services, such as DHCP and TFTP servers.
```

Comment:
```suggestion
You must configure the necessary network booting services, such as DHCP and TFTP.
```

suggestion (non-blocking): reword. Another idea: append "before you can provision a host".

I'd also drop "servers" because the DHCP/TFTP service can also just be part of any Smart Proxy.

If possible, would be good to link to https://docs.theforeman.org/nightly/Integrating_Provisioning_Infrastructure_Services/index-katello.html

Reply:
I've decided not to include the link here because I link to it in chapters for particular provisioning methods, ie. network boot.